src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  font-size: 30px;
+  padding: $square-paddings;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $square-spacing;
+  width: $calendar-width;
+
+  &__day {
+    @include square($square-size);
```

Comment:
The mixin `square($square-size)` is used here. Ensure that this mixin is defined in your SCSS files.